src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -57,33 +57,265 @@ Integer insertDataset(
         @Bind("dataUse") String dataUse,
         @Bind("dacId") Integer dacId);
 
+    @RegisterBeanMapper(value = User.class, prefix = "u")
     @UseRowReducer(DatasetReducer.class)
-    @SqlQuery(
-        "SELECT d.*, k.key, dp.property_value, dp.property_key, dp.property_type, dp.schema_property, dp.property_id, " +
-            " ca.consent_id, c.translated_use_restriction, dar_ds_ids.id as in_use, " +
-            FileStorageObject.QUERY_FIELDS_WITH_FSO_PREFIX +
```

Comment:
should we keep using the constant strings? now with use and fso both spelled out the query is very long / hard to parse, and maybe more brittle in case those change

Reply:
(same thought goes for all the below DAO changes)

Reply:
So ... I had a similar concern. I tried a couple different ways of incorporating the query string constants into text blocks and they all either didn't work syntactically or looked horrible. I went this route because it was much easier to run the dev/test cycle with an open sql editor than trying to finagle query strings between each run.